source/educators/how-tos/course_development/exercise_tools/create_poll_olx.rst

```diff
@@ -0,0 +1,106 @@
+.. _Create a Poll:
+
+Create a Poll for OLX
+########################
+
+.. tags:: educator, how-to
+
```

Comment:
I would add in the Note from the Reference to this page as well, because it's important.

```
.. note:: Creating a poll requires you to export your course, edit some of
 your course's XML files in a text editor, and then re-import your course. We
 recommend that you create a backup copy of your course before you create the
 poll. We also recommend that you only edit the files that will contain polls
 in the text editor if you are very familiar with editing XML.
```